src/client/components/Input/Input.stories.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Input from './Input';
+import { boolean, text, withKnobs } from '@storybook/addon-knobs';
+
+export default { title: 'Input', decorators: [withKnobs] };
+
+export const Inputfield = () => (
+  <Input
+    disabled={boolean('Disabled', false)}
```

Comment:
This property is passed to the component, but you never use it in the actual component.
Did you forget to use it or did you forget to remove it?